cardinal/config.go

```diff
@@ -12,6 +12,7 @@ const (
 	DefaultMode          = ModeDev
 	DefaultNamespace     = "world-1"
 	DefaultRedisPassword = ""
+	DefaultRedisAddress  = "redis:6379"
```

Comment:
i think this should stay localhost

Reply:
This will break the e2e tests. Unless you want the docker-compose to specify this.

Reply:
yeah lets have it specified in docker-compose

Reply:
The environment variable was already set. This is a bug. What was originally specified in docker-compose was REDIS-ADDR, but the code looks for REDIS_ADDRESS.